### ISSUE TITLE
Language Translation feature 

### ISSUE THREAD

**Issue by CBID2** [AccessibleForAll/AccessibleWebDev#265]
**Is your feature request related to a problem? Please describe.**
I'm concerned that non English Speakers who want to use the tools mentioned on the project's website won't be able to due to the content only being available in English.
**Describe the solution you'd like**
I'm thinking we can integrate non-code tool that provides a drop down menu where users can click their native language and the website automatically translates to said language. 

**Describe alternatives you've considered**
I've considered implementing this feature via code, but I realized that this could potentially mess up the website. 
**Additional context**
This is how I picture the feature to fit into the project's website:
<img width="1403" alt="Screenshot 2023-02-21 at 9 44 28 PM" src="https://user-images.githubusercontent.com/105683440/220507983-67fc0391-e972-4e6f-9f22-922f58133b8f.png">

**Comment by EmmaDawsonDev** [comment]
Thanks for the request Christine! The language support is actually already installed and ready to be developed (the homepage is even available in Swedish if you add /sv to the end of the url as I was testing it out) but I haven’t focused on it yet as I want to get all the content done in English. Then it’ll just be a case of getting things translated accurately.

**Comment by CBID2** [comment]
> Thanks for the request Christine! The language support is actually already installed and ready to be developed (the homepage is even available in Swedish if you add /sv to the end of the url as I was testing it out) but I haven’t focused on it yet as I want to get all the content done in English. Then it’ll just be a case of getting things translated accurately.

Oh ok. Have you thought of making this option in a physical widget like the image I presented?

**Comment by EmmaDawsonDev** [comment]
Yes, but only when enough of the site is translated so languages will be added to it one by one. It isn’t available yet as I don’t want to offer a translate button that only translates half the page when clicked.

**Comment by CBID2** [comment]
> Yes, but only when enough of the site is translated so languages will be added to it one by one. It isn’t available yet as I don’t want to offer a translate button that only translates half the page when clicked.
Oh I see @EmmaDawsonDev! 😊 Keep me posted

**Comment by kamranzafar4343** [comment]
I think language can be easily translated by using google translate extension. But the suggestion is also good as some people don't have extensions installed.
